tools/boxstarter/README.md

```diff
@@ -0,0 +1,54 @@
+# Boxstarter setup for Node.js
+
+A [Boxstarter][] script can be used for an easy setup of Windows systems with all
+the required prerequisites for Node.js development. This script will install
+the following [Chocolatey] packages:
+  * [Git for Windows][] with the `git` and Unix tools added to the `PATH`
+  * [Python 2.x][]
+  * [Visual Studio 2017 Build Tools][] with [Visual C++ workload][]
+  * Optionally: [Visual Studio Code][] e.g. when setting up Code and Learn boxes
+
+## WebLauncher Installation
+
+To install Node.js prerequisites using [Boxstarter WebLauncher][], just open
+**one** of  the following links with Internet Explorer or Edge browser on the
```

Comment:
Why does it need to be just those two browsers?

Reply:
It is a "Click Once" app (https://msdn.microsoft.com/en-us/library/t71a733d.aspx). It works in IE and Edge by default, other browsers need a plugin.